### ISSUE TITLE
Ability to enable/disable all Jet Engine features

### ISSUE THREAD

**Issue by Michaelgimii** [CrocoBlock/suggestions#6057]
**Please, describe the feature request, making it clear to understand.**

Hi crocoblock team!

Currently, JetEngine offers the ability to disable or enable some features. Like cct which is disabled by default but we can enable it. But it would be great if Jet Engine adds this feature to all its modules.
For example, I want to use only the metabox feature of Jet Engine. I enable it and disable other features such as cpt, listing, relations, taxonomies, options, etc. This will ensure optimal performance and also clean the environment of the jet engine.
Overall, it would be great if there was an enable/disable option for all jet engine features.

**Comment by 79ho3ein** [comment]
+1

**Comment by Mrgreen93** [comment]
Yup, its totally good idea. Also "metabox plugin" has this feature to enable/disable all of the features
Also their plugin is so light weight, i think this is the reason

**Comment by sekhsoyebali** [comment]
+1

**Comment by mehdimoradi1500** [comment]
Hi, so excited for this request @kaskad88
Each user base on the builder is using, should be able to disable or enable the JE's widgets/features
For instance, im using bricksbuilder with JE, so i don't need Gutenberg and elementor widgets of JE! So i should be able to disable them
Please consider this note too
